crates/edr_provider/src/debug_mine.rs

```diff
@@ -0,0 +1,74 @@
+use core::fmt::Debug;
+use std::sync::Arc;
+
+use edr_eth::Bytes;
+use edr_evm::{
+    state::{StateDiff, SyncState},
+    trace::Trace,
+    ExecutionResult, LocalBlock, MineBlockResultAndState, SyncBlock,
+};
+
+/// A trait for subscribing to `console.log` calls.
+pub trait ConsoleLogCallback: Debug {
```

Comment:
I think this can be removed

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4781/commits/4dc2638b1b77e757733abf63340d6ff75f882eb2